### ISSUE TITLE
Does this Container support switching listening port to VPN forwarded port?

### ISSUE THREAD

**Issue by Austin-2** [DyonR/docker-qbittorrentvpn#180]
Hi,

I want to switch over to this from Transmission (to support indexer specific seeding times) but I have one hesitation:

Does this support changing the listening port in preferences to whatever port your VPN uses? I use PIA, and can have a random port that is forwarded to me, so I want to know if this will support a changing port.

Thanks

**Comment by MYanello** [comment]
Yes, there is a setting within the qbittorrent ui or in the config file as 
```
[BitTorrent]
Session\Port=<port>
```

**Comment by stunrelay** [comment]
Hi. I’m in the exact same situation as OP. Also considering the switch. Is this feature of switching to the open port automatic? Or does this need to be manually defined at each reconnection to the VPN?

**Comment by MYanello** [comment]
How does this work for you now in Transmission? I would imagine you'd need some custom scripting to get the port via PIA's API if they give you a different one each re-connect. If you have that set up in Transmission you can likely use their script to get the port and place it in the Qbittorrent config file whenever the container restarts.
